plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -5608,6 +5608,23 @@ public void testCreateTableAsSelectFromVersionedTable()
         assertUpdate("DROP TABLE " + timestampVersionedSinkTableName);
     }
 
+    @Test
+    public void testSubqueryContainVersionedTable()
+    {
+        String tableName = "test_subquery_versioned" + randomNameSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT 1 id", 1);
```

Comment:
nit : Can we use `TestTable` here

Reply:
I would keep as-is for the consistency in this class.